containers/daap-python-base/src/var/task/versioning.py

```diff
@@ -200,6 +207,21 @@ def update_metadata_remove_schemas(self, schema_list: list[str]) -> str:
                 logger=self.logger,
             ).run()
 
+            # Copy data files in the curated bucket
```

Comment:
I thought this wasn't needed because it was already handled by the athena query in CuratedDataCopier. 

Rather than copy the all the parquet files in s3 we have an unload query that creates the new parquet from the latest load timestamp in the previous version. Am I misunderstanding how this works?

```sql
           UNLOAD (
                SELECT
                    *
                FROM {previous_major_database}.{curated_table.name}
                WHERE load_timestamp = (
                    SELECT MAX(load_timestamp)
                    FROM {previous_major_database}.{curated_table.name}
                )
            )
            TO '{self.table_path}'
            WITH(
                format='parquet',
                compression = 'SNAPPY',
                partitioned_by=ARRAY['load_timestamp']
            )
```

trigger from https://github.com/ministryofjustice/data-platform/blob/main/containers/daap-python-base/src/var/task/curated_data/curated_data_loader.py#L215